src/App.tsx

```diff
@@ -1,8 +1,88 @@
-import React from 'react';
-import './App.css';
+import React, { FC, useState } from 'react';
+import { TodoList } from './TodoList/TodoList';
+import { TodoProps, UserProps, UsersTodo } from './types';
+import { getUsers, getTodos } from './api/api';
 
-const App = () => (
-  <h1>Dynamic list of TODOs</h1>
-);
+const App: FC = () => {
+  const [todos, setTodos] = useState<UsersTodo[]>([]);
+  const [filteredTodos, setFilteredTodos] = useState<UsersTodo[]>([...todos]);
+  const [isLoading, setLoading] = useState(false);
+
+  const loadTodos = async () => {
+    setLoading(true);
+
+    const [todosFromServer, users] = await Promise.all([getTodos(), getUsers()]);
+
+    const preparedTodos = todosFromServer.map((todo: TodoProps) => {
+      const user = users.find((person: UserProps) => person.id === todo.userId) as UserProps;
```

Comment:
слишком длинная строка
```suggestion
      const user = users.find((person: UserProps) => (
        person.id === todo.userId
      )) as UserProps;
```